tests/test_advanced_filters.py

```
@@ -105,21 +105,21 @@ async def test_logical_operators_and_filters(
     found_names = [item["id"] for item in data["items"]]
     expected_names = [peer_names[i] for i in expected_peer_indices]
 
-    assert len(found_names) == len(expected_names), (
-        f"Expected {len(expected_names)} peers for {description}, got {len(found_names)}"
-    )
+    assert len(found_names) == len(
+        expected_names
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)